.github/ISSUE_TEMPLATE/Policy_package_proposal.md

```diff
@@ -1,35 +0,0 @@
----
-name: New built-in policy proposal
-about: If you have a new policy proposal that you think should be created.
----
-<!--
-    Thank you for contributing!
-
-    ISSUE TITLE:
-    Please prefix the issue title with a proposed policy name,
-    e.g. 'PolicyName: New policy proposal'
-
-    ISSUE DESCRIPTION (this template):
-    Please propose the new policy package under each header below.
--->
-
-### Description
-
-Tell the story about what you would like Policy to do
-
-### Proposed monitors
-
-- Audit something 1
-- Audit something 2
-
-### Proposed parameters
-
-- `[Type]` **ParameterName**:  The description for the parameter.
-
-### Package details if Guest Configuration
```

Comment:
Is the Guest Configuration team aware we're removing this?
(Is this form linked anywhere from documentation, where perhaps it should be removed?)